lib/test/fixtures/webpack-manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "js/webpack-runtime.js": "/blockly/js/webpack-runtime.338c9dacc207fe9b1eb4.min.js",
+  "js/vendors.js": "/blockly/js/vendors.b54aa8aa165e50793d97.min.js",
```

Comment:
this manifest fixture is borrowed from a future PR, which is why it contains content hashes. the code in this PR would generate a manifest which looks more like this:
```
{
  "js/webpack-runtime.js": "/blockly/js/webpack-runtime.min.js",
  "js/vendors.js": "/blockly/js/vendors.min.js",
```